### ISSUE TITLE
[BUG] The `latest` tag of the `apalache/mc` Docker image points to v0.8.0 instead of v0.15.2

### ISSUE THREAD

**Issue by romac** [informalsystems/apalache#739]
## Description

The [`latest` tag](https://hub.docker.com/layers/apalache/mc/latest/images/sha256-4d30ee0e9545cf904fbd6870d6fcee9a86e5c5961f57292641d6f89e80e7a2cb?context=explore) of the [`apalache/mc` Docker image](https://hub.docker.com/r/apalache/mc) points to version 0.8.0 instead of the latest released version, ie. 0.15.2.

## Input specification

N/A

## The command line parameters used to run the tool

```
docker run apalache/mc:latest version
```

## Current behavior

```
0.8.0-RELEASE
```

## Expected behavior

```
0.15.2 build v0.15.2-0-geb20fdf
```

## System information

- Apalache version [`apalache-mc version`]: See above
- OS [e.g. Ubuntu Linux or Mac OS, and the current version]: macOS 11.2.3
- JDK version [e.g. OpenJDK 0.8.0]: OpenJDK 64-Bit Server VM GraalVM CE 21.0.0 (build 25.282-b07-jvmci-21.0-b06, mixed mode)

## Additional context

I [noticed this problem](https://github.com/informalsystems/ibc-rs/pull/824/commits/11a96f84d8ae8efe8b224d16f3995a43805f8c62) when trying to use the Docker image to type check the IBC TLA+ specifications with GitHub Actions

**Comment by romac** [comment]
Made some edits above, ie. to use the `version` command instead of the program header.

**Comment by shonfeder** [comment]
Thanks for the report and sorry for the inconvenience!

Dockerhub was still targeting `master` for building the `latest` tag. It's now based off of the version tags, so it should be correct on our release next week.

I've also corrected the latest tag to point to 0.15.2: https://hub.docker.com/repository/docker/apalache/mc

**Comment by konnov** [comment]
That may explain why we got some reports about 0.8.0 :-)
